terraform/openstack/hosts-floating/main.tf

```diff
@@ -37,7 +38,7 @@ resource "openstack_blockstorage_volume_v1" "mi-control-lvm" {
 resource "openstack_blockstorage_volume_v1" "mi-resource-lvm" {
   name = "${ var.short_name }-control-lvm-${format("%02d", count.index+1) }"
   description = "${ var.short_name }-control-lvm-${format("%02d", count.index+1) }"
-  size = "${ var.data_volume_size }"
+  size = "${ var.resources_data_volume_size }"
```

Comment:
@ryane yeah, good point. I'll refactor this code tomorrow and ensure that workers are called workers in whole Mantl :-)